submissions/cynthiaNwakaeme/php-database/connection.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+     //connecting to mysql database
+
+     try {
+        $pdo = new PDO('mysql: host=localhost', 'root', 'mysql');

```

Comment:
How are you passing the database name here? I can only see hostname, username and password

Reply:
I used the syntax "Use db_name"(SQL SYNTAX) in the file where I am creating the database and table